eureka/S5_lightcurve_fitting/fitters.py

```diff
@@ -928,17 +1011,33 @@ def load_old_fitparams(meta, log, channel, freenames):
 
     return np.array(fitted_values)[0]
 
-def save_fit(meta, lc, model, fitter, fit_params, freenames, samples=[], upper_errs=[], lower_errs=[]):
-    # Save the fitted parameters and their uncertainties (if possible)
+
+def save_fit(meta, lc, model, fitter, results_table, freenames, samples=[], spec_table=None):
+    """Save fitted parameters and chains
+    
+    Notes
+    -----
+    History:
+    - Mar 13-Apr 18, 2022 Caroline Piaulet
+         Record an astropy table for mean, median, percentiles, +/- 1 sigma, all params
+         Save transmission spectrum (commented out)
+    """
     if lc.share:
         fname = f'S5_{fitter}_fitparams_shared'
     else:
         fname = f'S5_{fitter}_fitparams_ch{str(lc.channel).zfill(len(str(lc.nchannel)))}'
-    data = fit_params.reshape(1,-1)
-    if len(upper_errs)!=0 and len(lower_errs)!=0:
-        data = np.append(data, -lower_errs.reshape(1,-1), axis=0)
-        data = np.append(data, upper_errs.reshape(1,-1), axis=0)
-    np.savetxt(meta.outputdir+fname+'.csv', data, header=','.join(freenames), delimiter=',')
+    results_table.write(meta.outputdir+fname+'.csv', format='csv', overwrite=False)
```

Comment:
Given that the file format for the S5 file has changed, you will also need to change the way in which that file is read in by S6

Reply:
done now!

Reply:
done now!